aiida/work/utils.py

```diff
@@ -138,10 +152,14 @@ def exponential_backoff_retry(fct, initial_interval=10.0, max_attempts=5, logger
             else:
                 logger.warning('iteration %d of %s excepted, retrying after %d seconds', iteration + 1, coro.__name__,
                                interval)
-                yield sleep(interval)
+
+                import traceback
+                traceback.print_exc()
```

Comment:
Do we always want the print here? It will be dumped once it hits the exception handling of the process no?